src/frontend/ai-reviewer-admin/src/App.js

```diff
@@ -14,9 +15,9 @@ function App() {
   return (
     <>
       <Header header={header} />
-      <div className="content">
```

Comment:
This actually breaks the sticky footer.
App.scss has
.content {
  flex: 1 0 auto;
}
which makes the footer "stick" to the bottom when the page is shorter than the viewport.  The middle body auto expands to fill 100% vertical.

Reply:
Ah kk will revert. Thanks:)

Reply:
or incorporate the content css class into the Container wrapper

Reply:
Great. this works again.